name: 'Audeliano Li'
hierarchy_level: 2
login_name: 'audeliano'
password: 'teste1234567'
location: {x: -0.01, y: 5, theta: deg(9.0)}
computer: {hostname: 'audeliano_pc', mobile: false, location: {x: -0.01, y: 5, theta: deg(9.0)}}
